ports/servoshell/Cargo.toml

```diff
@@ -119,17 +117,18 @@ http = { workspace = true }
 net = { path = "../../components/net" }
 net_traits = { workspace = true }
 serde_json = { workspace = true }
+# For optional feature servo_allocator/use-system-allocator
+servo_allocator = { path = "../../components/allocator" }
```

Comment:
I don't understand what this comment is trying to tell me. Does the crate not do anything if the feature is disabled? This does not seem like info that needs to be in `Cargo.toml`, since `servo_allocator` is included unconditionally.

Reply:
It looks like that comment was added in #31443 by @sagudev. Should it be removed?

Reply:
To be fair, the comment was already present, it just got moved in this PR.